CHANGELOG.md

```diff
@@ -4,6 +4,28 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v8.0.2](https://github.com/voxpupuli/puppet-bacula/tree/v8.0.2) (2025-01-27)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-bacula/compare/v8.0.0...v8.0.2)
+
+**Implemented enhancements:**
+
+- Adding an option for the bacula file daemon to support custom message types [\#219](https://github.com/voxpupuli/puppet-bacula/pull/219) ([mi-reichel](https://github.com/mi-reichel))
+- Add support for FreeBSD 14 [\#211](https://github.com/voxpupuli/puppet-bacula/pull/211) ([smortex](https://github.com/smortex))
+- Improve data types [\#193](https://github.com/voxpupuli/puppet-bacula/pull/193) ([smortex](https://github.com/smortex))
```

Comment:
Due to the enhancements, this should be a minor version bump (8.1.0), not a patch version bump.